### PR TITLE
Changes to make nodes/api work with latest Dingo API

### PR DESCRIPTION
**Pull request by pcoutinho**
This will allow Nodes/API to work with the latest Dingo API version. I also changed the composer.json to use dingo/api locked to the following commit: dev-master#8cd1a6cd7d2ba9fc96841fd0745ef8fc1ce2a530 

Please review and test before making a release.

### CODE REVIEW

**Comment by rugaard** [comment]
Have you gone through all the 36 commits that `dingo/api` was ahead or only fixed the router part? It's quite important that we get **all** the stuff that's changed. It's all good that a request works, but a lot of time we've lost other features because we didn't review all commits :)

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     ],
     "require": {
         "laravel/framework": "5.1.*||5.2.*",
-        "nodes/dingo": "^1.0",
+        "dingo/api": "dev-master#8cd1a6cd7d2ba9fc96841fd0745ef8fc1ce2a530",
```
Comment (rugaard) [inline composer.json]: We should never lock the package to a specific commit hash. That's why we forked the repository in the first place. Instead we need to update the forked repository :)

**Comment by rugaard** [comment]
If you can remove your force of commit hash from `composer.json` it all looks good. Just went through all the commits from `dingo/api` and looks like you either went through all of them or were "lucky" (:P) about there not being any other significant changes :)

Also in the future, it would be cool if you - since you're a part of the Nodes team - could make your PR's in the develop branch of the repository. This way we can all use your branch and we don't merge from an external master branch to our internal master branch. We're suckers for having collected in our develop branches, so it looks "correct" in our list of PR's - small stuff, but danish people (read: especially me) has a weird kind of OCD ;D

I've updated our `nodes/dingo` repository to contain all the latest `dingo/api` changes. So once we merge this package, we'll make a new release of the forked repo and then make a new release of this package :)

**Comment by pcoutinho** [comment]
Going to do it the right way.

**Pull request closed without merging**